Sage-Bionetworks/BridgePF
# Misc small fixes

Fixes:
- signing in to disabled accounts now throws a 423 instead of a 503
- signIn and verifyEmail now display Stormpath token, study ID, and internal session token in the logs
- https://sagebionetworks.jira.com/browse/BRIDGE-1047 - signing in when already signed in now properly checks consent status
- https://sagebionetworks.jira.com/browse/BRIDGE-1065 - hack to convert timestamps into calendar dates
- as a side effect of these changes, the Auth Controller now checks Consent status and throws 412, not the Auth Service

Updated unit tests and ran unit tests for all affected files.

Manual Tests:
- sign up
- verify email, verify 412, check metrics in logs, sign out
- sign in, verify 412, check metrics in logs
- sign in again (from cached session), verify 412, check metrics in logs
- sign out, add developer role to account
- sign in, verify 200
- sign in again (from cached session), verify 200
- sign out, remove developer role from account, disable account
- sign in, verify 423
- sign out, re-enable account
- sign in, consent, sign out
- sign in, verify 200
- sign in again (from cached session), verify 200
- add schema with calendar date, upload data with timestamp instead of calendar date, verify that it passes strict validation
